### PR TITLE
Simple registration

### PR DESCRIPTION
**Pull request by Cezikos**

### CODE REVIEW

File: src/javascripts/pages/register/register-page.js [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+  constructor(props : Props) {
+    super(props);
+
+    this.state = {
+      email: '',
+      username: '',
+      password: '',
+      firstname: '',
+      surname: '',
+      city_id: 1,
```
Comment (falseinput) [inline src/javascripts/pages/register/register-page.js]: city_id -> cityId

File: src/javascripts/pages/register/register-page.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+  _validator = () => {
+    if(this.state.username.length < 2) {
+      this.setState({
+        error: STRINGS.REGISTER.TOO_SHORT_USERNAME
+      });
+      return false;
+    }
+
+    if(EmailValidator.validate(this.state.email) !== true) {
```
Comment (falseinput) [inline src/javascripts/pages/register/register-page.js]: Please use regexp instead ;)
https://stackoverflow.com/a/1373724

File: src/javascripts/pages/register/register-page.js [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+  _register = () => {
+    if(this._validator() === true) {
+      UserApi.register({
+        email: this.state.email,
+        username: this.state.username,
+        password: this.state.password,
+        firstname: this.state.firstname,
+        surname: this.state.surname,
+        city_id: this.state.city_id
```
Comment (falseinput) [inline src/javascripts/pages/register/register-page.js]: city_id -> cityId

File: src/javascripts/pages/register/register-page.js [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+          this.props.navigation.navigate('Main');
+        })
+        .catch(err => {
+          console.log(err);
+
+        });
+      })
+      .catch(err => {
+        console.log(err);
+        var obj = JSON.parse(err._bodyText);
```
Comment (falseinput) [inline src/javascripts/pages/register/register-page.js]: Wouldn't plain`err.status` work? Without parsing?

File: src/javascripts/pages/register/register-page.js [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+              value={this.state.password}
+              onChangeText={this._onChangeText('password')}
+              placeholder={STRINGS.REGISTER.PASSWORD}
+              underlineColorAndroid="transparent"
+              secureTextEntry={true} />
+            <PrimaryButtonSmall
+              onPress={this._register}
+              customStyles={styles.registerButton}
+              text={STRINGS.REGISTER.SIGNUP} />
+
```
Comment (falseinput) [inline src/javascripts/pages/register/register-page.js]: Unnecessary whitespace

File: src/javascripts/pages/register/register-page.js (hunk 1 above)
Comment (Cezikos) [reply src/javascripts/pages/register/register-page.js]: In db we have city_id

File: src/javascripts/pages/register/register-page.js (hunk 2 above)
Comment (Cezikos) [reply src/javascripts/pages/register/register-page.js]: Done

File: src/javascripts/pages/register/register-page.js (hunk 4 above)
Comment (Cezikos) [reply src/javascripts/pages/register/register-page.js]: With `err._bodyText` I have specified problem

File: src/javascripts/pages/register/register-page.js (hunk 5 above)
Comment (Cezikos) [reply src/javascripts/pages/register/register-page.js]: Done
